Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -48,9 +48,12 @@ There are usually two release engineers working at any given time. It goes witho
   1. Input the new version number.
     
 During this stage, the release manager has the following tasks:
-  1. Create the CRP ticket by triggering the Slack command (eg. `/crp ios branch:release/babylon/4.1.0`) in `#ios-launchpad`		
-    * This will also generate the CHANGELOG automatically (from the list of commits between the `release/{appname}/{version}` branch used in the command and the tag for the latest version of the same product – i.e. the most recent `{appname}/*` tag) to include it in the CRP ticket		
-    * Further manually complete the CRP ticket with any additional information (clinical risk, etc)	
+
+  1. Ensure the CRP ticket is up-to-date
+
+    * The CRP ticket is created automatically for iOS during release cutoff ([see details here](https://github.com/babylonhealth/babylon-ios/blob/develop/Documentation/Process/Release%20process/CRP-Bot.md))
+    * But they need to ensure all tickets have had their Fix Version field updated as expected (see the CRP report)
+    * They also need to manually complete the CRP ticket with some additional information (clinical risk, etc)	
```

Comment:
In GitHub's rich diff view the formatting is off for those points. Not sure if it's actual issue.
<img width="951" alt="image" src="https://user-images.githubusercontent.com/830743/73855832-8bd56380-482c-11ea-8d35-7d239ceab7ec.png">

Reply:
Good eye sir 👍 
Tried a couple of space/indentation/newline combination, and finally managed to find one that seems to be working